src/js/lib/Scrollspy.jsx

```diff
@@ -142,36 +144,42 @@ export class Scrollspy extends React.Component {
   }
 
   _handleSpy () {
-    let timer
```

Comment:
It was to throttle scroll events but seems not work. Maybe I had misunderstood throttle function. i will add function to throttle scroll event soon.